lite/examples/object_detection/android/app/src/main/java/org/tensorflow/lite/examples/detection/tflite/TFLiteObjectDetectionAPIModel.java

```diff
@@ -230,11 +233,17 @@ public String getStatString() {
   public void close() {}
 
   public void setNumThreads(int num_threads) {
-    if (tfLite != null) tfLite.setNumThreads(num_threads);
+    // if (tfLite != null) tfLite.setNumThreads(num_threads);
+    if (tfLiteOptions != null) {
+      tfLiteOptions.setNumThreads(num_threads);
+    }
   }
 
   @Override
   public void setUseNNAPI(boolean isChecked) {
-    if (tfLite != null) tfLite.setUseNNAPI(isChecked);
+    // if (tfLite != null) tfLite.setUseNNAPI(isChecked);
```

Comment:
The only issue here is that you'd want to recreate the interpreter to have the options take effect.